### ISSUE TITLE
Enchantment table does not work in 1.17.

### ISSUE THREAD

**Issue by lewismercer** [ItsSchatten/PortableCraftingInvs#20]
The etable gui opens up but when I put ann item in to the slot it doesnt let you enchant it. Does this work your end or is it an issue my side?

Thanks,
Lewis

**Comment by ItsSchatten** [comment]
Yes I can confirm that the enchantment table doesn't seem to allow enchanting in 1.17, though I'm not entirely sure as to why at this time.

I'll look into getting this fixed asap.

**Comment by lewismercer** [comment]
Awesome, thanks for the speedy response!
________________________________

**Comment by ItsSchatten** [comment]
Quick little update to this:

I've exhausted my knowledge of messing with the enchant table in NMS and in the Spigot API. I've also posted a thread asking the community for help with this and if I'm able to get a working enchantment table working I'll release an update for it.

Unfortunately this means that enchantment tables will not function in the plugin until I either find a work around or get what I currently have working.

I apologies for the any inconvenience this may have caused, I'm going to leave this issue open until it can be resolved completely.
